@ornikar/jest-config/test-shim.js

```diff
@@ -1,6 +1,8 @@
 'use strict';
 
 /* Always load Intl Polyfill */
+require('@formatjs/intl-pluralrules/polyfill');
+require('@formatjs/intl-pluralrules/dist/locale-data/fr');
```

Comment:
d'après la doc, ca serait plutot `full-icu` qu'il faudrait utiliser, en remplacement de @formatjs/intl-pluralrules et intl ? https://github.com/formatjs/react-intl/blob/master/docs/Getting-Started.md#nodejs 
Avec un TODO pour remove quand on passera à node 14 LTS ?

Reply:
`If your node version is missing any of the Intl APIs above, you'd have to polyfill them accordingly.`

C'est pas forcément lié non ? c'est plus de savoir si y a le support du pluriel dans le node actuel

Reply:
ah oui je n'ai pas lu cette phrase

Reply:
ca serait pas mieux de mettre apres l'import l'intl du coup ?

Reply:
vu que IntlPolyfill va overrider global.Intl non ?